alpaca/trading/stream.py

```diff
@@ -30,12 +30,12 @@ def __init__(
         secret_key: str,
         paper: bool = True,
         raw_data: bool = False,
-        url_override: str = None,
+        url_override: Optional[str] = None,
         websocket_params: Optional[Dict] = None,
     ):
         self._api_key = api_key
         self._secret_key = secret_key
-        self._trade_updates_handler = None
+        self._trade_updates_handler: Optional[Callable] = None
```

Comment:
Perhaps, to be a bit more specific, we can specify this function's expected input and output:
```suggestion
        self._trade_updates_handler: Optional[Callable[[Union[BaseModel, RawData]], None]]= None
```
And same for the arg in the function signature for `subscribe_trade_updates`.